src/Tribe/Tickets.php

```diff
@@ -2302,6 +2345,8 @@ public function process_attendee_meta( $product_id, $meta ) {
 		 *
 		 * @internal
 		 *
+		 * @throws ReflectionException
```

Comment:
:no_entry_sign: Comment missing for @throws tag in function comment

<sub>hash: 53b0628baee50280c310bb7c5a5498b4</sub>